roles/klaytn_grafana/tasks/grafana.yml

```diff
@@ -1,32 +1,40 @@
 # description: Setup grafana
-- name: "Make configuration directory"
+- name: "Make data and configuration directory"
   file:
     path: "{{ item }}"
     state: "directory"
   with_items:
-    ["{{ grafana_conf_dir }}/provisioning/datasources", "{{ grafana_conf_dir }}/provisioning/dashboards"]
+    ["{{ grafana_conf_dir }}/provisioning/datasources", "{{ grafana_conf_dir }}/provisioning/dashboards", "{{ influxdb_data_dir }}"]
   become: yes
 
+- name: "Generate random strings for InfluxDB password"
```

Comment:
If I want to get the randomly generated password to work the database by login, how can it achved?

Reply:
Use one of below 3 options.
1. See generated `telegraf.conf`
2. See generated `klaytn.yml`
3. In the InfluxDB docker, check environment variable`INFLUXDB_USER_PASSWORD`

Reply:
Stupid question. Thank you